doc/nrf/known_issues.rst

```diff
@@ -916,6 +916,16 @@ NCSDK-12020: Current consumption for Gaming Mouse increased by 1400mA
 
   **Workaround:** Change ``pwm_pin_set_cycles`` to ``pwm_pin_set_usec`` in function :c:func:`led_pwm_set_brightness` in Zephyr's driver :file:`led_pwm.c` file.
 
+nRF Machine Learning
+====================
+
+.. rst-class:: v1-9-0
+
+NCSDK-13923: Device may crash when Bluetooth Central establishes security
```

Comment:
```suggestion
NCSDK-13923: Device may crash when Bluetooth Central establishes secured connection
```
Would this work? "Established security" sounds like slang. Also, why "may"? What are the conditions on which this "may" depends? Can the stack size be large enough for some supported devices? Can it increase with some Kconfig options?

Reply:
The secure connections in Bluetooth refers to option for pairing (LE Secure Connections). I will rephrase it a bit.

Reply:
I think the stack may overflow, but it seems like there are also cases when connection security is properly established. It depends on operations that are actually performed over Bluetooth and implementation, I would not go into details here .